change/@fluentui-react-menu-8a8164e2-5df5-4f72-8183-03ba92fb1427.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28559&commitId=568db2d2eaae92a572bd7d6888d2429834177ec6&env=prod&clientType=FLUENTUI)


  <details><summary><b>AvatarGroup Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|AvatarGroup Converged.overflowContent - Dark Mode.popoverContentOpen.chromium.png|9271| Changed|
|AvatarGroup Converged.overflowContent.popoverContentOpen.chromium.png|3584| Changed|
|AvatarGroup Converged.overflowContent - High Contrast.popoverContentOpen.chromium.png|11564| Changed|

</details>


  <details><summary><b>Menu</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu.Nested Submenus Small Viewport Flipped.nested menu.chromium.png|734| Changed|
|Menu.Nested Submenus Small Viewport Stacked.nested menu.chromium.png|695| Changed|

</details>


  <details><summary><b>Menu Converged - MenuItemLinks</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - MenuItemLinks.default.hover menuitemlink.chromium.png|41| Changed|
|Menu Converged - MenuItemLinks.default.chromium.png|46| Changed|

</details>


  <details><summary><b>Menu Converged - basic</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - basic.default - RTL.hover menuitem.chromium.png|41| Changed|
|Menu Converged - basic.default - RTL.chromium.png|46| Changed|
|Menu Converged - basic.default.chromium.png|46| Changed|
|Menu Converged - basic.default.hover menuitem.chromium.png|41| Changed|

</details>


  <details><summary><b>Menu Converged - groups</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - groups.default - Dark Mode.chromium.png|2307| Changed|
|Menu Converged - groups.default - RTL.chromium.png|121| Changed|
|Menu Converged - groups.default - High Contrast.chromium.png|2739| Changed|
|Menu Converged - groups.default.chromium.png|121| Changed|

</details>


  <details><summary><b>Menu Converged - nested submenus</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - nested submenus.default - RTL.chromium.png|387| Changed|
|Menu Converged - nested submenus.default.all open.chromium.png|712| Changed|
|Menu Converged - nested submenus.default - RTL.all open.chromium.png|1044| Changed|
|Menu Converged - nested submenus.default.chromium.png|64| Changed|

</details>


  <details><summary><b>Menu Converged - selection</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - selection.checkbox - Dark Mode.chromium.png|919| Changed|
|Menu Converged - selection.checkbox.chromium.png|46| Changed|
|Menu Converged - selection.checkbox - High Contrast.selected.chromium.png|1046| Changed|
|Menu Converged - selection.checkbox - Dark Mode.selected.chromium.png|920| Changed|
|Menu Converged - selection.checkbox - RTL.chromium.png|46| Changed|
|Menu Converged - selection.checkbox.selected.chromium.png|41| Changed|
|Menu Converged - selection.checkbox - High Contrast.chromium.png|1082| Changed|
|Menu Converged - selection.checkbox - RTL.selected.chromium.png|41| Changed|

</details>


  <details><summary><b>Menu Converged - selection groups</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - selection groups.default - RTL.chromium.png|362| Changed|
|Menu Converged - selection groups.default.chromium.png|117| Changed|
|Menu Converged - selection groups.default - RTL.selected.chromium.png|358| Changed|
|Menu Converged - selection groups.default.selected.chromium.png|111| Changed|

</details>


  <details><summary><b>Menu Converged - split item</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - split item.default - RTL.chromium.png|85| Changed|
|Menu Converged - split item.default - RTL.submenu open.chromium.png|1047| Changed|
|Menu Converged - split item.default.chromium.png|84| Changed|
|Menu Converged - split item.default.submenu open.chromium.png|1053| Changed|

</details>


  <details><summary><b>Menu Converged - submenuIndicator slotted content</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - submenuIndicator slotted content.default - RTL.chromium.png|95| Changed|
|Menu Converged - submenuIndicator slotted content.default - RTL.submenus open.chromium.png|1353| Changed|
|Menu Converged - submenuIndicator slotted content.default.submenus open.chromium.png|1369| Changed|
|Menu Converged - submenuIndicator slotted content.default.chromium.png|95| Changed|

</details>


  <details><summary><b>Menu nested within a MenuTrigger</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu nested within a MenuTrigger.default.submenu open.chromium.png|839| Changed|
|Menu nested within a MenuTrigger.default.chromium.png|50| Changed|

</details>


  <details><summary><b>Popover Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|913| Changed|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|221| Changed|
|Popover Converged.positioning.positioned popovers.chromium.png|439| Changed|
|Popover Converged.avoid scrolling.PopoverSurface focused.chromium.png|10485| Changed|

</details>


  <details><summary><b>Positioning</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.auto size.chromium.png|6743| Changed|

</details>


  <details><summary><b>Shadow DOM</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Shadow DOM.Portal.normal.chromium.png|72| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>